pipelines/curate_rewards.py

```
@@ -50,104 +50,216 @@
+    "reward_signature",
+    "shutil",
+    "sys",
+    "units_migration_catalog",
+    "validate_conversion_policy",
     "validate_ontology_document",
-)
+    "value_type",
+]
+
+
+def _entry_calibrations(entry, *, path, index):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_entry_calibrations`

(ANN202)